app/main.py

```diff
@@ -83,11 +83,14 @@ def grab_secrets(self):
         output_folder = os.getenv('SECRETS_FOLDER')
 	secrets_output_folder = os.path.join(output_folder, "secrets")
 	certs_output_folder = os.path.join(output_folder, "certs")
+        keys_output_folder = os.path.join(output_folder, "keys")
 
         if not os.path.exists(secrets_output_folder) :
```

Comment:
I think it would be cleaner to simply iterate over the folders, e.g.:
```python
for folder in (secrets_output_folder, certs_output_folder, keys_output_folder):
    if not os.path.exists(folder) :
        os.makedirs(folder)
```

Reply:
fixed